history.go

```diff
@@ -55,7 +55,9 @@ func (r *Recorder) EnsureHistoryFileOpen() error {
 func (r *Recorder) Session() {
 	err := r.EnsureHistoryFileOpen()
 	if err != nil {
-		fmt.Fprint(r.Stdout, err)
+		r.Stop()
+		fmt.Fprintln(r.Stdout, err)
```

Comment:
This should go to standard error, shouldn't it?

Reply:
good catch! thx.